examples/pytorch/README.md

```diff
@@ -251,3 +252,87 @@ or if in a Conda environment:
 ```bash
 conda install -c comet_ml -c anaconda -c conda-forge comet_ml
 ```
+
+### Neptune
+
+1. Install the Neptune client library:
+
+    pip:
+
+    ```bash
+    pip install neptune-client
+    ```
+
+    conda:
+
+    ```bash
+    conda install -c conda-forge neptune-client
+    ```
+
+1. Import `NeptuneCallback`:
+
+    ```python
+    from transformers.integrations import NeptuneCallback
+    ```
+
+1. Enable Neptune logging in your script:
+
+    - In your `TrainingArguments`, set the `report_to` argument to `"neptune"`:
```

Comment:
No need for another list here, the text is good as it is and all the nestedness doesn't render super well.

Reply:
Done I think